static/scripts/rewards/gift-cards/mint/mint-action.ts

```diff
@@ -32,48 +34,48 @@ export function attachMintAction(giftCard: GiftCard, app: AppState) {
 }
 
 async function mintGiftCard(productId: number, app: AppState) {
-  if (app.signer) {
-    const country = await getUserCountryCode();
-    if (!country) {
-      toaster.create("error", "Failed to detect your location to pick a suitable card for you.");
+  if (!app.signer) {
+    toaster.create("error", "Connect your wallet.");
+    return;
+  }
+
+  const country = await getUserCountryCode();
+  if (!country) {
+    toaster.create("error", "Failed to detect your location to pick a suitable card for you.");
```

Comment:
Shouldn't we default to the international one

Reply:
The international one supports 139 countries. Knowing the country is important so that we don't offer card to the rest of the countries in the world. It will be a loss for them to buy a card they cannot redeem or use.